frontend/src/components/EntityList.vue

```diff
@@ -171,14 +120,14 @@ export default {
       filters: {},
       orderBy: "name",
       ascending: true,
+      loading: true
     };
   },
+  mounted() {
+    if (this.schema !== undefined && this.schema !== null) {
```

Comment:
Can't this be simplified to `if (this.schema)`?🤔

Reply:
Yes, we can simplify it to `if (!this.schema)` :wink: